TEAMWORK.md

```diff
@@ -23,11 +25,14 @@ Each team member will work on their assigned task individually. This approach en
 **Debugging and Error Fixing**: We will allocate time after pull request submission to address any issues or bugs identified during the review process. It is crucial to avoid last-minute submissions.
 
 ## Communication
-**Tools**: Slack will serve as our real-time communication channel, while GitHub Issues will be used for tracking and discussing project-related subjects.
+
+**Tools**: Slack will serve as our real-time communication channel, while GitHub will be used for tracking and discussing project-related subjects.
 
 **Slack Usage**: Slack will primarily be utilized for quick queries, updates, and general discussions.
 
-**GitHub Issues Usage**: GitHub Issues will be employed for more in-depth discussions, bug tracking, and task assignments. We commit to responding to GitHub Issues within 12 hours on weekdays.
+**GitHub Usage**: GitHub Pull Requests are the place where we talk about suggested code changes. Following guidelines apply:
```

Comment:
I feel like we should discuss suggestions in an issue or PR and come to an agreement and then assign a person to implement the actual change